Dockerfile

```diff
@@ -9,15 +9,16 @@ RUN mkdir -p /var/lib/ironic-inspector && \
 
 RUN crudini --set /etc/ironic-inspector/inspector.conf DEFAULT auth_strategy noauth && \ 
     crudini --set /etc/ironic-inspector/inspector.conf ironic auth_strategy noauth && \
-    crudini --set /etc/ironic-inspector/inspector.conf DEFAULT debug true && \
+    crudini --set /etc/ironic-inspector/inspector.conf DEFAULT debug false && \
     crudini --set /etc/ironic-inspector/inspector.conf database connection sqlite:///var/lib/ironic-inspector/ironic-inspector.db && \
     crudini --set /etc/ironic-inspector/inspector.conf DEFAULT transport_url fake:// && \
     crudini --set /etc/ironic-inspector/inspector.conf processing store_data database && \
+    crudini --set /etc/ironic-inspector/inspector.conf processing ramdisk_logs_dir /shared/log/ironic-inspector/ramdisk && \
+    crudini --set /etc/ironic-inspector/inspector.conf processing always_store_ramdisk_logs true && \
```

Comment:
In case we haven't done it already, we should pass `ipa-debug=1` to the ramdisk.

Reply:
Yes, we are setting that in inspector.ipxe - https://github.com/metalkube/metalkube-ironic/blob/master/inspector.ipxe#L7